packages/core/android/src/test/kotlin/com/datadog/reactnative/DdSdkTest.kt

```diff
@@ -1827,8 +1827,10 @@ internal class DdSdkTest {
         @StringForgery initializationType: String,
         @BoolForgery trackInteractions: Boolean,
         @BoolForgery trackErrors: Boolean,
-        @BoolForgery trackNetworkRequests: Boolean
-    ) {
+        @BoolForgery trackNetworkRequests: Boolean,
+        @StringForgery reactVersion: String,
```

Comment:
minor: the following regex can be used for version `regex = "[1-9]{1,3}\\.[1-9]{1,3}\\.[1-9]{1,3}"`

but in this case it doesn't really matter.